Enable Compile for Shard Tensor (#1682)
NVIDIA/physicsnemo: examples/cfd/external_aerodynamics/transformer_models/README.md

> **Note** Like other parameters of the model, changing the value of `model.use_te` will make checkpoints incompatible.

> The training script supports data-parallel training via PyTorch DDP. In a future update, we may enable domain parallelism via ShardTensor (with DDP or FSDP2 `fully_shard` on the data-parallel axis).

Reviewer: ```suggestion
The training script supports data-parallel training via PyTorch DDP. A possible future change might enable domain parallelism usingShardTensor (with DDP or FSDP2 `fully_shard` on the data-parallel axis).
```
